composer.json

```diff
@@ -145,6 +146,8 @@
         "spryker/discount-promotions-rest-api": "^1.4.2",
         "spryker/documentation-generator-rest-api": "^1.15.0",
         "spryker/dummy-marketplace-payment": "^0.2.3",
+        "spryker/dynamic-entity-backend-api": "^1.0.0",
+        "spryker/dynamic-entity-gui": "^1.0.0",
```

Comment:
Should be integrated as part of the feature, see https://release.spryker.com/features/view/155